test/test_lyrics.py

```diff
@@ -163,7 +163,7 @@ def test_remove_credits(self):
 
         # don't remove 2nd verse for the only reason it contains 'lyrics' word
         text = """Look at all the shit that i done bought her
-                  See lyrics ain't nothin
+                  See lyrics ain't nothing
```

Comment:
This may be an artistic _nothin_

Reply:
Yeah it is. Don't change the test please.